src/Microsoft.AspNetCore.SignalR.Redis/RedisHubLifetimeManager.cs

```diff
@@ -383,7 +383,7 @@ private void SubscribeToAll()
                 {
                     RedisLog.ReceivedFromChannel(_logger, _channels.All);
 
-                    var invocation = _protocol.ReadInvocation(data);
+                    var invocation = _protocol.ReadInvocation((byte[])data);
```

Comment:
I see, this is because of RedisValue

Reply:
Yeah, it has conversions to `byte[]` and `string`, not `ReadOnlyMemory<byte>`... yet... ;)